packages/utils/src/period/getDefaultPeriod.js

```diff
@@ -6,7 +6,8 @@
 import { convertDateRangeToPeriods } from './convertDateRangeToPeriods';
 import { utcMoment } from '../datetime';
 
-export const EARLIEST_DATA_DATE = utcMoment('2017-01-01'); // Tupaia started in 2017
+export const EARLIEST_DATA_DATE_STRING = '2017-01-01'; // Tupaia started in 2017
```

Comment:
Interesting to be reminded of this...completely outside the scope of this PR but we do now have PSSS data stretching back to 2015, though I guess if we need to show it on tupaia.org @enunan will just need to create a ticket for extending the earliest date supported on tupaia :-)

Reply:
Good to know! Perhaps this kind of information should live in the DB (and be configurable via the admin panel)

Reply:
Yeah that could work, like an earliest date per project setting. For now we're still ok with it as is, so will resolve this conversation